recipes/example/meta.yaml

```diff
@@ -61,23 +65,25 @@ test:
 
 about:
   home: https://github.com/simplejson/simplejson
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+  description: |
+    simplejson is a simple, fast, complete, correct and extensible
+    JSON <https://json.org> encoder and decoder for Python 2.5+ and
+    Python 3.3+. It is pure Python code with no dependencies, but includes
+    an optional C extension for a serious speed boost.
   # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
   # Use the SPDX identifier, e.g: GPL-2.0-only instead of GNU General Public License version 2.0
   # See https://spdx.org/licenses/
   license: MIT
-  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". 
+  # Optional
   license_family: MIT
-  # It is strongly encouraged to include a license file in the package,
+  # It is required to include a license file in the package,
```

Comment:
cc @conda-forge/core on this specific change

Reply:
Yes I wasn't sure about this one. We don't have it written down anywhere but it has been de-facto standard now. Might need a CFEP.

Reply:
I don't think we need a CFEP. I are already enforcing a hard requirement for a license file. This is from old days where so many packages did not have one. I think it is fine to keep the change.